server/routes/order.router.js

```diff
@@ -0,0 +1,136 @@
+const express = require('express');
+const pool = require('../modules/pool');
+const router = express.Router();
+const {
+  rejectUnauthenticated,
+} = require('../modules/authentication-middleware');
+
+/**
+ * GET route template
+ */
+router.get('/', rejectUnauthenticated, async (req, res) => {
+  try {
+    const queryText = `
+    SELECT * FROM "orders" 
+    WHERE orders."companyID" = $1 
+    ORDER BY ("testingStatus" = 'Pre-shipment') DESC;`;
+    const dbRes = await pool.query(queryText, [req.user.companyID]);
+    res.send(dbRes.rows);
+  } catch (err) {
+    console.error(err.message);
+    res.sendStatus(500);
+  }
+});
+
+router.put('/newOrder/:id', async (req, res) => {
```

Comment:
We're not quite following REST API naming conventions here. 

Here's what I'm seeing in this router:
- GET /api/orders
- POST /api/orders/initialOrder
- PUT /api/orders/newOrder/:id
- PUT /api/orders/newOrder
- PUT /api/orders/shipping

Just looking at the code, it's not super clear what the difference is between an `initialOrder` and a `newOrder`. Or, when would we use `PUT /newOrder/:id` vs. `PUT /newOrder` (with orderId in the body vs `PUT /shipping` (which also updates the order, but only with shipping info?)

Ideally, our API would look like:
- GET /api/orders
- POST /api/orders
- PUT /api/orders/:id

...and that `PUT` endpoint would be flexible enough to handle different combinations of request body values. 

Those PUT endpoints, especially, can be a little tricky to make that flexible. Not expecting a change here, just dropping some wisdom bombs on y'all.